### PR TITLE
Issue/index containers

### LINKED ISSUE #12: Index file for Containers and Actions

### LINKED ISSUE #15: Add a linter to project
Add a way to format the files and check for corrections

### LINKED ISSUE #16: Use enviroments variables to APIs

### PR DESCRIPTION
**Pull request by TitusCln**
Fix #12 Close #15 Closes #16

### CODE REVIEW

**Comment by oscarSantoyo** [comment]
The linked issue is to add indexed exports, but the code changes more than that...

**Comment by TitusCln** [comment]
Should i add another issue and link it?

**Comment by oscarSantoyo** [comment]
@TitusCln Yes please

File: src/containers/userstories/AddUserstory.js [hunk 1]
```diff
@@ -1,65 +1,80 @@
...
-    addUserStory:(projectId,newUserstory)=>dispatch(addUserStory(dispatch,projectId,newUserstory))
-})
+  projectId: state.global.projectId,
+});
+const mapDispatchToProps = (dispatch) => ({
+  addUserStory: (projectId, newUserstory) =>
+    dispatch(addUserStory(dispatch, projectId, newUserstory)),
+});
 
 //TODO CAMBIAR LOS REFS POR OBJECTOS(TAGS) NATIVOS DE REACT
```
Comment (oscarSantoyo) [inline src/containers/userstories/AddUserstory.js]: Remove these comments, create issues instead

File: src/index.js [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
+import "./index.css";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
+import reducers from "./reducers";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 
-const store=createStore(reducers,window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__())
+const store = createStore(
+  reducers,
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```
Comment (oscarSantoyo) [inline src/index.js]: This extension should be only active in DEV env

File: src/reducers/global.js [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-    switch(action.type){
-        case 'SET_CURRENT_PROJECT':
-            return Object.assign({},state,{
-                touched: true
-            })
-        default:
-            return state
-    }
-}
+const global = (state = {}, action) => {
```
Comment (oscarSantoyo) [inline src/reducers/global.js]: This file is not used, we should remove it.

File: src/setupProxy.js [hunk 4]
```diff
@@ -1,21 +1,22 @@
...
 const app = express();
 
-module.exports = function(app) {
+module.exports = function (app) {
   app.use(
-    '/apiprojects',
+    "/apiprojects",
     createProxyMiddleware({
-      target: 'http://localhost:8090',
+      target: "http://localhost:8090",
```
Comment (oscarSantoyo) [inline src/setupProxy.js]: Please create a proxy for prod env, or a way to read this value from ENV var.

File: src/setupProxy.js [hunk 5]
```diff
@@ -1,21 +1,22 @@
...
   );
   app.use(
-    '/apiuserstories',
-    createProxyMiddleware({target:'http://localhost:8080',
-    changeOrigin:true,
-    pathRewrite:{'/apiuserstories':''}
-  })
+    "/apiuserstories",
+    createProxyMiddleware({
+      target: "http://localhost:8080",
```
Comment (oscarSantoyo) [inline src/setupProxy.js]: Same here: Please create a proxy for prod env, or a way to read this value from ENV var.

**Review by oscarSantoyo — CHANGES_REQUESTED** [review]
Great job congratulations, I leave some comments :medal_military:

**Comment by TitusCln** [comment]
@oscarSantoyo  i've already review and work your comments

File: .gitignore [hunk 6]
```diff
@@ -44,3 +44,6 @@ yarn-error.log*
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+#enviroments variables
+.env
```
Comment (oscarSantoyo) [inline .gitignore:49]: could you add a file `.env.example` where you explain the variables that are suppose to go in `.env`, please?

File: src/index.js [hunk 7]
```diff
@@ -8,10 +8,7 @@ import * as serviceWorker from "./serviceWorker";
 import reducers from "./reducers";
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 
-const store = createStore(
```
Comment (oscarSantoyo) [inline src/index.js]: But if you remove these lines we are not able to use Redux dev tools, my point was to only make then available for DEV and not for production

**Review by oscarSantoyo — CHANGES_REQUESTED** [review]
Left a pair of comments, but really good work :dark_sunglasses:

**Review by oscarSantoyo — APPROVED** [review]
Good job!!

**Pull request merged**